src/OMSimulatorLib/System.cpp

```diff
@@ -1669,6 +1677,40 @@ oms_status_enu_t oms::System::delete_(const oms::ComRef& cref)
   return oms_status_error;
 }
 
+oms_status_enu_t oms::System::deleteStartValue(const ComRef& cref)
+{
+  oms::ComRef tail(cref);
+  oms::ComRef front = tail.pop_front();
+
+  if (tail.isEmpty())
+  {
+    if (oms_status_ok != startValues.deleteStartValue(front))
+    {
+      logWarning("failed to delete start value \"" + std::string(getFullCref()+cref)+":start" + "\"" + " because the identifier couldn't be resolved to any System");
```

Comment:
typo

Reply:
and should we really return `oms_status_ok`?

Reply:
Yes we need to return oms_status_ok, because, otherwise the logWarning at the end of function 
 ` logWarning("failed to delete start value \"" + std::string(getFullCref()+cref)+":start" + "\"" + " because the identifier couldn't be resolved to any connector, component, system, or model");
`
will be printed for all the values, and this warning should be printed only for `unknown ` system or component which you can see in testcase